components/styles/styler.component.ts

```diff
@@ -25,21 +25,8 @@ type StyleJson = {
 })
 export class HsStylerComponent {
   icons: any;
-  imagetypes: Array<any> = [
```

Comment:
Originally this approach was used to support translations of "Icon"and "Circle". hrname could change, but underlying value would remain,.

Reply:
Ah, I did not realised that.. Will revert my "improvement" then.

Reply:
Reverted.